requirements.txt

```diff
@@ -1,198 +1,384 @@
-#### Requirements for running the server ####
-
-bleach==3.3.0 \
- --hash=sha256:6123ddc1052673e52bab52cdc955bcb57a015264a1c57d37bea2f6b817af0125 \
- --hash=sha256:98b3170739e5e83dd9dc19633f074727ad848cbedb6026708c8ac2d3b697a433
+bleach==3.3.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0") \
```

Comment:
Most of those changes seem unnecessary. I think we should only modify the 2 changed libraries according to `poetry.lock`